### PR TITLE
Temporary main menu background fix

### PR DESCRIPTION
**Pull request by CI09**
**Brief Description of What This PR Does**

It fixes *a bit* menu backgrounds, they are not ideal but more reminescent of pre-conversion ones.

**Related Issues**

**Progress Checklist**

Note: before starting this checklist the PR should be marked as non-draft.

- [x] PR author has checked that this PR works as intended and doesn't
      break existing features:
      https://wiki.revolutionarygamesstudio.com/wiki/Testing_Checklist
      (this is important as to not waste the time of Thrive team
      members reviewing this PR)
- [x] Initial code review passed (this and further items should not be checked by the PR author)
- [ ] Functionality is confirmed working by another person (see above checklist link)
- [ ] Final code review is passed and code conforms to the 
      [styleguide](https://github.com/Revolutionary-Games/Thrive/blob/master/doc/style_guide.md).

Before merging all CI jobs should finish on this PR without errors, if
there are automatically detected style issues they should be fixed by
the PR author. Merging must follow our
[styleguide](https://github.com/Revolutionary-Games/Thrive/blob/master/doc/style_guide.md#git).

### CODE REVIEW

File: assets/models/menu_backgrounds/MainMenuScene1.tscn [hunk 1]
```diff
@@ -354,9 +354,10 @@ environment = SubResource("1")
 
 [node name="starssphere" parent="." instance=ExtResource("2")]
 transform = Transform3D(10000, -2.45936, 1.74526, 2.44336, 9999.59, 91.1048, -1.76759, -91.1044, 9999.59, -12.7566, 0, 8.74701)
+visible = false
```
Comment (hhyyrylainen) [inline assets/models/menu_backgrounds/MainMenuScene1.tscn:357]: Why is this not visible?

File: assets/models/IronRock1.mesh
Comment (hhyyrylainen) [inline assets/models/IronRock1.mesh:1]: I assume these rock meshes look fine in the microbe stage still.

**Comment by hhyyrylainen** [comment]
The code / technical changes here look fine to me now. I haven't checked in game (and especially I haven't checked if the assets now look fine in the space stage as it loans many of the main menu 3D assets). But I think this could be merged if you think this is ready now.

**Comment by CI09** [comment]
I slowed down camera a bit, if it's fine then I guess it can be merged unless planet texture is green for some reason (it is on my local repo but I hope I didn't push any changes related to that, but someone confirming everything is alright would be cool)

**Comment by hhyyrylainen** [comment]
I'll merge this then, any future issues should be quickly visually obvious to people playing the new builds.

**Pull request merged**